api/v1alpha1/nstemplatetier_types.go

```diff
@@ -22,6 +22,18 @@ type NSTemplateTierSpec struct {
 	// +optional
 	// +mapType=atomic
 	SpaceRoles map[string]NSTemplateTierSpaceRole `json:"spaceRoles,omitempty"`
+
+	// SpaceRequestConfigName stores all the configuration related to the Space Request feature
+	// +optional
+	SpaceRequestConfigName *SpaceRequestConfig `json:"spaceRequestConfig,omitempty"`
```

Comment:
Why is there the "Name" suffix?
It should be
```suggestion
	// SpaceRequestConfig stores all the configuration related to the Space Request feature
	// +optional
	SpaceRequestConfig *SpaceRequestConfig `json:"spaceRequestConfig,omitempty"`
```